crates/e2e/macro/src/lib.rs

```diff
@@ -57,10 +57,10 @@ use syn::Result;
 /// You can switch the E2E test to use the [DRink!](https://use.ink/basics/contract-testing/drink)
 /// testing framework with this syntax:
 ///
-/// ```
+/// ```no_compile
```

Comment:
It doesn't have `ink_sandbox` as dependency so it doesn't compile

Reply:
You could make `ink_sandbox` a dev-dependency?